provider/src/test/kotlin/org/gradle/kotlin/dsl/integration/KotlinBuildScriptModelIntegrationTest.kt

```diff
@@ -232,7 +241,9 @@ class KotlinBuildScriptModelIntegrationTest : AbstractIntegrationTest() {
 
         assertContainsBuildSrc(classPath)
         assertContainsGradleKotlinDslJars(classPath)
-        assertExcludes(classPath, existing("classes.jar"))
+
```

Comment:
⭕️ I don't think this empty line is necessary but if you feel strongly about it...